build.sbt

```diff
@@ -13,7 +13,7 @@ lazy val root = project
   .in(file("."))
   .aggregate(profiledb, plugin, profilingSbtPlugin)
   .settings(
-    name := "profiling-root",
+    name := "scalac-profiling",
```

Comment:
That's not good. The `plugin` project already has `name := "scalac-profiling"`.

Reply:
🤦🏻  good catch, thank you.